### PR TITLE
[QT-20,26,53,55] update password change success and fail dialog & cen…

### PR DESCRIPTION
**Pull request by nikitaromanenko99**
https://blocksettle.atlassian.net/jira/software/c/projects/QT/issues/QT-20
https://blocksettle.atlassian.net/jira/software/c/projects/QT/issues/QT-26
https://blocksettle.atlassian.net/jira/software/c/projects/QT/issues/QT-53
https://blocksettle.atlassian.net/jira/software/c/projects/QT/issues/QT-55

### CODE REVIEW

File: GUI/QtQuick/qml/WalletProperties/ChangePassword.qml [hunk 1]
```diff
@@ -9,13 +9,30 @@ import "../StyledControls"
...
+        target:bsApp
+        function onSuccessChangePassword ()
+        {
+            layout.sig_success()
+        }
+    }
+
+    CustomMessageDialog {
+        id: error_dialog
+        error: qsTr("Password should be over 6 charaters")
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/WalletProperties/ChangePassword.qml]: typo (charaters)
Also "6" should be taken from the constant added earlier to QtQuickAdapter.cpp, or change the text to something like "password strength is insufficient" or "password failed to pass validation checks".

File: GUI/QtQuick/qml/WalletProperties/ChangePassword.qml [hunk 2]
```diff
@@ -135,4 +168,13 @@ ColumnLayout {
         new_password.input_text = ""
         confirm_password.input_text = ""
     }
+
+    function checkPasswordLength()
+    {
+        if (new_password.input_text.length < 6) {
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/WalletProperties/ChangePassword.qml]: Shouldn't you call `bsApp.validatePassword` here instead?

Comment (nikitaromanenko99) [reply GUI/QtQuick/qml/WalletProperties/ChangePassword.qml]: I will follow your opinion

File: GUI/QtQuick/qml/WalletProperties/ChangePassword.qml (hunk 1 above)
Comment (nikitaromanenko99) [reply GUI/QtQuick/qml/WalletProperties/ChangePassword.qml]: bsApp.validatePassword is used in password integrity check, so I will use "password strength is insufficient" as error message.

**Review by sergey-chernikov — APPROVED** [review]

**Comment by sergey-chernikov** [comment]
Resolve the conflict please

**Pull request merged**